src/components/HtmlImport/store/htmlImport.js

```diff
@@ -43,10 +140,25 @@ export default {
 
 			commit("toggleFetching");
 		},
+		async updateImportSettings({ getters, rootState }) {
+			const settings = getters.getImportConfigObject.settings;
+			while (rootState.ImportInited === "pending") {
+				await new Promise(resolve => setTimeout(resolve, 100));
+			}
+			if (rootState.ImportInited === true) {
+				Vue.prototype.$chamaileon.Import.methods.updateSettings(settings);
+			}
+		},
 	},
 	getters: {
+		getImportSettings(state) {
+			return state.settings;
+		},
```

Comment:
remove getImportSettings
also remove this: getJSONFetchStatus 
maybe remove: getImportBtns